resources/lang/fr.js

```diff
@@ -83,20 +83,57 @@ class CactbotLanguageFr extends CactbotLanguage {
       Pull: 'Pull',
     });
 
+    this.kPetNames = Object.freeze([
+      // Pulled from https://xivapi.com/Pet?pretty=true&language=fr
+      'Carbuncle émeraude',
+      'Carbuncle topaze',
+      'Ifrit-Egi',
+      'Titan-Egi',
+      'Garuda-Egi',
+      'Eos',
+      'Selene',
+      'Auto-tourelle Tour',
+      'Auto-tourelle Fou',
+      'Demi-Bahamut',
+      'Demi-Phénix',
+      'Séraphin',
+      'Carbuncle hécatolite',
+      'Estime',
+      'Automate Reine',
+      'Ombre',
+    ]);
+
     this.countdownStartRegex = function() {
-      return Regexes.parse(/Début du combat dans (\y{Float}) secondes !/);
+      return Regexes.gameLog({
+        capture: true,
+        line: 'Début du combat dans (?<time>\\y{Float}) secondes !',
+      });
     };
     this.countdownEngageRegex = function() {
-      return /:À l'attaque !/;
+      return Regexes.gameLog({
+        capture: true,
+        line: 'À l\'attaque !',
```

Comment:
Mmm, does this actually work? I know this was same bit was there before, but #57 had some discussion about the space before the !

Reply:
I think I understand why you have a problem with this space. as I do in the replace text translations, you have to put the two bars \\ to avoid the double symbol. if you don't put them there is a double symbol and the line doesn't work. Here's what I did to avoid these double symbols
- Gaoler Landslide\\?': 'Geôlier glissement ?',

Reply:
`!` is no regex symbol so it does not have to be escaped.
The issue with `À l\'attaque !` is that in the game files it is written like this: `À l'attaque<Indent/>!` where the `<Indent/>` is handeled differently in each language.
AFAIK in FR, `<Indent/>` is replaced with a single space while in DE it is just plain removed e.g. `Start!`

Reply:
Do you have an example game log line with the `À l'attaque !` text?

Reply:
In previous discussion we decided to remove the exclamation mark (!), it works fine without it (and did not work when included).

Here's what we get in log `00|2020-05-06T16:51:45.0000000+02:00|0039||À l'attaque!|da622a0378a60d251ae3348393625685` but we got `À l'attaque !` on screen and in the chatlog as you can see below.
![image](https://user-images.githubusercontent.com/40036185/81192363-6a664e00-8fba-11ea-9f40-2b69e3c03bb5.png)

Reply:
The issue is, switching to a `Regexes.gameLog` call for this always matches the entire line. So the correct syntax would be:
```javascript
      return Regexes.gameLog({
        capture: true,
        line: `À l'attaque[ ]?!`,
      });
```

This results in this regex:
```javascript
/(?<timestamp>^.{14}) 00:(?<code>....):(?<line>À l'attaque[ ]?!)$/i
```

Which when ran against your example network log produces:

![image](https://user-images.githubusercontent.com/6119598/81195053-56572800-8f8b-11ea-9ae7-52844809dcee.png)

Also tested with `[10:51:45.000] 00:0039:À l'attaque !`, that also matches.

Reply:
As mentioned below, there is a space in French. It appears in the game, but in the log it has no space.

![ffxiv_06052020_184843_764](https://user-images.githubusercontent.com/61565834/81205017-65a99600-8fca-11ea-99ea-65693e44ea14.png)

Reply:
> In previous discussion we decided to remove the exclamation mark (!), it works fine without it (and did not work when included).

Thanks for the clarification.  I had forgotten what we had decided.  That means that this existing regex above was wrong after all (but not for the reason I thought).